src/library/waveform.js

```diff
@@ -0,0 +1,57 @@
+// This is the library for creating the waveform
+class Waveform {
+
+  /** This is the constructor for initializing waveform
+   * @param {String} canvasId The HTML id where waveform is located.
```

Comment:
I would use the term "element" rather than `HTML`.

So, I would say - An ID of a canvas element where the waveform will be rendered.